### ISSUE TITLE
tab style: selected not distinguishable

### ISSUE THREAD

**Issue by andreas-p** [opnsense/core#723]
e.g. in firewall rules, the currently selected tab isn't highlighted; the selection is only visible if a mouse is hovered over the tabs. The current selection should be indicated clearly.

**Comment by fichtner** [comment]
Which browser/OS is this? There should not be a problem in all recent major browsers.

**Comment by andreas-p** [comment]
Firefox 43+44 and Safari 9.0.3 on OSX 10.10.: all white.

Just checked on Linux/Gnome3: tab is highlighted white vs. very light blue; might still be a little more distinctive because the difference is quite small, esp. on a notebook display if viewed with some angle.

**Comment by fichtner** [comment]
This falls in the domain of custom layout / colouring to fix it. Some combinations (or VGA monitors) won't render this properly apparently, but it is not the norm.

**Comment by andreas-p** [comment]
I can follow this when talking about the notebook display issue, but on Mac OSX apparently a system color that opnsense assumes to be non-white actually is white by default.

**Comment by fichtner** [comment]
This is Chrome and Safari on OSX:

<img width="1251" alt="screen shot 2016-02-16 at 9 15 42 am" src="https://cloud.githubusercontent.com/assets/1915288/13070623/3aa23cb0-d48e-11e5-957b-3eb03efea7ee.png">
<img width="1067" alt="screen shot 2016-02-16 at 9 16 16 am" src="https://cloud.githubusercontent.com/assets/1915288/13070625/3d74d6f0-d48e-11e5-9557-dda78e4a6060.png">

Are we talking about different things?

**Comment by andreas-p** [comment]
No, that's what I've been talking about: LAN selected but not highlighted.

**Comment by fichtner** [comment]
Why should it be highlighted? It's already active.

I've checked browser tabs in Chrome, Firefox and Safari, all of them have this behaviour: highlight on hover for inactive tabs, but not active ones.

If indeed there is a highlight that is "invisible" then that is a bit convoluted but intended so.

**Comment by andreas-p** [comment]
Ok I just made a screenshot on Linux/Gnome, and sent that image to my OSX. On the linux notebook, highlighting of tabs was visible, and the menu shows three shades of white: unselected, selected section (e.g. firewall) and selected item (e.g. rules). That screenshot only shows the 'rules' menu selection, so I examined it using Gimp. Apparently, white is color FFFFFF, and lighter greys that vanish on my osx machine have F5F5F5 (menu) or F7F7F7. the selected item uses E8E8E8.

So this indeed boils down to a personal monitor issue; after playing around with some fancy Samsung "Magic Bright" setting the monitor shows F7F7F7 as non-white now.
Sorry for the noise, never missed those super bright grey tones...
